app-ios/Core/Sources/Model/NotificationSettings.swift

```diff
@@ -0,0 +1,35 @@
+import Foundation
+
+public struct NotificationSettings: Equatable, Sendable {
+    public let isEnabled: Bool
+
+    public let reminderMinutes: Int
```

Comment:
IMO: Since NotificationReminderTime exists, it might be good to have a stronger association or impose some constraints related to it.